sympy/parsing/tests/test_latex.py

```diff
@@ -187,7 +193,10 @@ def test_import():
 def test_parseable():
     from sympy.parsing.latex import parse_latex
     for latex_str, sympy_expr in GOOD_PAIRS:
-        assert parse_latex(latex_str) == sympy_expr
+        try:
+            assert parse_latex(latex_str) == sympy_expr, (latex_str, sympy_expr)
+        except AssertionError: # if the assertion fails, show what triggered the error
+            raise AssertionError(latex_str + str(sympy_expr))
```

Comment:
It isn't necessary to catch the AssertionError and then raise it again. I think the original code here was fine.

Reply:
I imagine this is one of those cases where local testing with pytest works fine, but that lack of pytest in CI makes this awkward